probe-rs/src/debug/debug_info.rs

```diff
@@ -497,15 +498,20 @@ impl DebugInfo {
         &self,
         core: &mut Core<'_>,
         address: u64,
+        exception_info: &Option<ExceptionInfo>,
         unwind_registers: &registers::DebugRegisters,
     ) -> Result<Vec<StackFrame>, DebugError> {
         let mut units = self.get_units();
 
-        let unknown_function = format!(
-            "<unknown function @ {:#0width$x}>",
-            address,
-            width = (unwind_registers.get_address_size_bytes() * 2 + 2)
-        );
+        let unknown_function = if let Some(exception_info) = exception_info {
+            exception_info.description.to_string()
+        } else {
+            format!(
+                "<unknown function @ {:#0width$x}>",
+                address,
+                width = (unwind_registers.get_address_size_bytes() * 2 + 2)
```

Comment:
It would be nice to have a comment for this magic formula.

Reply:
Done :)